deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,52 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+.eerste {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+.tweede {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+.item {
+    flex-basis: 33.33%
```

Comment:
Nice, deze had ik dus met flex: 1; Maar zo is die van jou ook responsive en schaalt hij mee!

Reply:
In de uitwerkingen gaan ze dus uit van een vaste aantal pixels, wat ik zelf een beetje suf vind.